src/main/resources/avro/variantmethods.avdl

```diff
@@ -67,17 +67,17 @@ record SearchVariantsRequest {
   /**
   The `VariantSet` to search.
   */
-  string variantSetId;
+  string variantSetId = "";
 
   /**
   Only return variant calls which belong to call sets with these IDs.
   If an empty array, returns variants without any call objects.
   If null, returns all variant calls.
   */
-  union { null, array<string> } callSetIds = null;
+  union { null, array<string> } callSetIds;
```

Comment:
Oops, I actually didn't mean to modify this at all in this PR. Per the PR description, this field is meant to be resolved in #488.